subsys/bluetooth/host/hci_core.h

```diff
@@ -271,7 +271,11 @@ struct bt_le_per_adv_sync {
 };
 
 struct bt_dev_le {
-	/* LE features */
+	/** @brief Remote LE features
+	 *
+	 * Available after `atomic_test_bit(conn->flags, BT_CONN_LE_FEATURES_EXCHANGED)`.
+	 * Signaled by bt_conn_cb.remote_info_available().
+	 */
 	uint8_t			features[8];
```

Comment:
those are local features,   this comment should be in struct bt_conn_le instead, right?